governance-exercises/Polling Exercise #2 Response.md

```diff
@@ -0,0 +1,40 @@
+#### My Interpretation of the MIPs and Plan
```

Comment:
As a sidenote. These are explicitly polling exercises. I appreciate that you've concluded there is no need for a poll here, but just for the sake of practice, you could have included one based on a different set of assumptions.